### PR TITLE
test: (e2e) add carousel e2e tests

### PR DESCRIPTION
**Pull request by Seamoo13**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/4376

#### Please provide a brief summary of this pull request.
Add e2e tests for the Carousel component.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by nikvalor — APPROVED** [review]

File: e2e/wdio/core/tests/carousel.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        hideCardBtns,
+        hiddenPageIndicator,
+        hiddenNavBtns,
+        numberedPagination,
+        errorMsg,
+        busyIndicator,
+        disableLoadingBtn,
+        contentNavBtns,
+        sectionTitle
+    } = new CarouselPo();
```
Comment (valorkin) [inline e2e/wdio/core/tests/carousel.e2e-spec.ts:39]:
```suggestion
    } = carouselPage;
```

**Review by valorkin — APPROVED** [review]
lgtm, just one comment

File: e2e/wdio/core/tests/carousel.e2e-spec.ts (hunk 1 above)
Comment (Seamoo13) [reply e2e/wdio/core/tests/carousel.e2e-spec.ts]: done

Comment (valorkin) [reply e2e/wdio/core/tests/carousel.e2e-spec.ts]: thx

**Review by AntonOlkhovskyi — APPROVED** [review]

**Pull request merged**